mPower2/MotorControl/Resources/Finger_Tapping.json

```diff
@@ -0,0 +1,222 @@
+                    }
+                },
+                "viewTheme":{
+                    "viewIdentifier":"Instruction",
+                    "storyboardIdentifier":"ActiveTaskSteps"
+                },
+                "actions":{
+                    "goForward":{
+                        "buttonTitle": "Start the test"
+                    }
+                },
+             
+             },
+             {
+                 "identifier":"left",
+                 "type":"skipableSection",
+                 "asyncActions":[
+                                 {
+                                     "identifier":"motion",
+                                     "type":"motion",
+                                     "startStepIdentifier":"countdown",
+                                     "requiresBackgroundAudio":true,
+                                     "recorderTypes":["accelerometer", "gyro"]
+                                 }
+                                ],
+                 "steps":[
+                          {
+                              "identifier":"leftActive",
+                              "type":"active",
+                              "duration":5,
+                              "image":{
+                                  "imageName":"TapLeft1",
+                                  "placementType":"fullsizeBackground"
+                              },
+                              "colorTheme":{
+                                  "usesLightStyle":true
+                              },
+                              "viewTheme":{
+                                  "viewIdentifier":"Action",
+                                  "storyboardIdentifier":"ActiveTaskSteps"
+                              },
+                              "commands":["shouldDisableIdleTimer", "vibrate", "playSound"],
+                              "spokenInstructions":{
+                                  "start":"Tap the buttons with your LEFT hand"
+                              }
+                           }
+                        ]
+             },
+             {
+                 "identifier":"right",
+                 "type":"skipableSection",
+                 "asyncActions":[
+                                 {
+                                     "identifier":"motion",
+                                     "type":"motion",
+                                     "startStepIdentifier":"countdown",
+                                     "requiresBackgroundAudio":true,
+                                     "recorderTypes":["accelerometer", "gyro"]
+                                 }
+                                ],
+                 "steps":[
+                          {
+                              "identifier":"rightActive",
+                              "type":"active",
+                              "duration":5,
+                              "image":{
+                                  "imageName":"TapRight1",
```

Comment:
Won't the image be flipped by your image flipping code?

Reply:
This isn't implemented yet for the active steps, so I will make that change when that gets implemented.